Sources/BraintreeCore/Public/BraintreeCore/BTConfiguration.h

```diff
@@ -6,6 +6,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Contains information specific to a merchant's Braintree integration
  */
+// Would it make sense to collapse all of the Configuration+ categories into a single class here?
```

Comment:
We looked into this, and we actually like how the configuration categories are organized now. We think having all the properties in one class might be too much.

We did open a PR to add an `environment` property to `BTConfiguration`, though: #603